docs/source/issues.rst

```diff
@@ -5,18 +5,16 @@ Known issues
 
 .. _elb_batch_len_setting:
 
-ELB_BATCH_LEN setting
----------------------
+Batch length setting
+--------------------
 
 The value of :ref:`ELB_BATCH_LEN` greatly affects performance. We are in the
 process of determining reasonable defaults for various programs.
 
-.. _disk_leak:
+.. _suboptimal_performance:
 
-Persistent disk is not cleaned up
```

Comment:
I would still leave a note about possible persistent disk leak, maybe saying that it is unlikely, but can happen. Especially if users keep on hitting Ctrl-C anything can happen.

Reply:
Sounds good, I added this in d1a7ae9.